src/services/project/project_test.go

```diff
@@ -75,3 +75,36 @@ func TestCorrectRuntime(t *testing.T) {
 		t.Fatalf(fmt.Sprintf("Runtime %v not properly replaced", proj_runtime.RuntimeVersion))
 	}
 }
+
+func TestInit(t *testing.T) {
+
+	supportedPipelineTypes := []string{"github"}
+
+	for _, pipelineType := range supportedPipelineTypes {
+		sourcePipelineTemplate := path.Join(root, "assets", pipelineType, "pipeline.tmpl")
+	
+		// check if embedded folder and pipeline.tmpl file exists
+		if _, err := os.Stat(sourcePipelineTemplate); err != nil {
+			t.Fatalf(fmt.Sprintf("Error: template file for supported pipeline %v dont exists", pipelineType))
+		}
+
+		testDestinationFolder := fmt.Sprintf(".init_test_%s", pipelineType)
+
+		// test function
+		_, err := ProjectInit(pipelineType, testDestinationFolder, os.DirFS(root))
+
+		if err != nil {
+			t.Fatalf(fmt.Sprintf("Error: %v", err))
+		}
+
+		// remove file created
+		if err = os.Remove(path.Join(testDestinationFolder, fmt.Sprintf("%s_pipeline.yaml", pipelineType))); err != nil {
```

Comment:
We should probably have the file path returned by the Project package, it's not worth it replicate the same logic, it might cause issues.

Reply:
Maybe something like init returning `[]string` will all files created instead of returning a message that can be printed by the code itself?

Reply:
Do you mean update the test code to return a list of string? Right now, it only returns a single file

If is just for test, do we need to have the file path in the project package?

Reply:
Updates made